tools/docker/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+/usr/local/bin/naive_snapshotter \
+     -address /var/run/firecracker-containerd/naive-snapshotter.sock \
+     -path /var/lib/firecracker-containerd/naive \
+     -debug &> /var/log/naive_snapshotter.out &
```

Comment:
Are we exposing `/var/log/naive_snapshotter.out` and `/var/log/containerd.out` somewhere?  These will be useful to see in test failures.

Reply:
> need to ensure each test run is carved out its own directory on the host which is eventually cleaned up after a while

If you write to the current working directory, the Buildkite Agent will (attempt) to clean it up on exit.  (The "attempt" is because the Buildkite Agent does not run as root, and will get tripped up by file permissions.)

Reply:
Got this working, including uploading as buildkite artifacts! Thanks!